pyproject.toml

```diff
@@ -79,6 +80,23 @@ classifiers = [
 ]
 
 [project.optional-dependencies]
+build = [
```

Comment:
This seems conceptually wrong, these are duplicate from the `requires` table. And `dev.py` dependencies are `dev` not `build`.